src/scss/layouts/_header.scss

```
@@ -0,0 +1,81 @@
+header {
+  background: rgba($color: $color-white, $alpha: 0.1);
+  @include absolute();
```

```suggestion
  @include absolute();
  background: rgba($color: $color-white, $alpha: 0.1);
```
`@include` cứ đưa hết lên trên đầu nhé